src/sentry/api/endpoints/organization_events.py

```diff
@@ -233,20 +233,9 @@ def handle_data(self, request, organization, project_ids, results):
         if not ("project.id" in first_row or "projectid" in first_row):
             return results
 
-        projects = {
-            p["id"]: p["slug"]
-            for p in Project.objects.filter(organization=organization, id__in=project_ids).values(
-                "id", "slug"
-            )
-        }
         for result in results:
             for key in ("projectid", "project.id"):
                 if key in result:
-                    # Handle bizarre empty case
```

Comment:
Why are you removing this? Did you fix the case where we get an empty results row?

Reply:
Ah -- sorry this was from me rebasing from master-- but I think its fixed. The third arg to transform `"transform(project_id, [{}], {}, '')"` is what to do if there's an unknown value. so it should set `project.name=""` there for us.

Do you know how to recreate this /or if there's a test for this for me to confirm?

Reply:
nvm added a test for this: https://github.com/getsentry/sentry/pull/16988/files#diff-c45ca977527fcfa0b27a7e26485664cfR114